src/plot/Optimal_Meatballs.jl

```diff
@@ -0,0 +1,1248 @@
+### A Pluto.jl notebook ###
+# v0.19.36
+
+#> [frontmatter]
+#> license_url = "https://unlicense.org/"
+#> title = "Optimal Meatballs"
+#> date = "2024-02-08"
+#> tags = ["circle-packing", "Plots.jl", "PlutoUI.jl"]
+#> description = "Learn about circle packing by optimizing the size of meatballs!  🍝"
+#> license = "Unlicense"
+#> 
```

Comment:
Adding a cover image would be nice, [maybe this image?](https://commons.wikimedia.org/wiki/File:Polpettes_et_linguini_Pasta_%C3%A9_Pizza_(Lyon)_mars_2023_(2).jpg). It has a CC0 licence, so you can freely reuse it here.
```suggestion
#> image = "https://upload.wikimedia.org/wikipedia/commons/thumb/f/fb/Polpettes_et_linguini_Pasta_%C3%A9_Pizza_%28Lyon%29_mars_2023_%282%29.jpg/540px-Polpettes_et_linguini_Pasta_%C3%A9_Pizza_%28Lyon%29_mars_2023_%282%29.jpg"
#>
```

Reply:
Yes! I forgot to add a cover image.